src/main/java/blackjack/domain/result/DealerResult.java

```diff
@@ -0,0 +1,42 @@
+package blackjack.domain.result;
+
+import blackjack.domain.common.Name;
+
+import java.util.EnumMap;
+import java.util.List;
+
+public class DealerResult {
+    private final Name name;
+    private final EnumMap<ResultStatus, Count> resultStatusBoard;
+
+    public DealerResult(final Name name, final EnumMap<ResultStatus, Count> resultStatusBoard) {
+        this.name = name;
+        this.resultStatusBoard = resultStatusBoard;
+    }
```

Comment:
아래처럼 구현하면 EnumMap 구현체의 이점은 누리면서 유연한 코드를 작성할 수 있겠네요.

```suggestion
    private final Map<ResultStatus, Count> resultStatusBoard;

    public DealerResult(final Name name, final Map<ResultStatus, Count> resultStatusBoard) {
        this.name = name;
        this.resultStatusBoard = new EnumMap<>(resultStatusBoard);
    }

```